hasura/graphql-engine
# console: select first operator by default (close #5729)

### Description
When filtering rows in the data tab in the hasura console, the first operator will be selected by default.

### Changelog


not yet, I'm not sure under which version I need to add this.

### Affected components
- Console

### Related Issues
#5729

### Solution and Design
![image](https://user-images.githubusercontent.com/1710840/92306812-4a5b9100-ef92-11ea-8081-c0b4127af52a.png)

### Steps to test and verify
* login to the console
* go to the data tab
* select a table
* see the the first filter has the first operator selected by default. This will probably be _eq in most setups. 

#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?

#### GraphQL

#### Breaking changes
